examples/btc_subscription.pl

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+no indirect;
+
+use IO::Async::Loop;
+use Data::Dumper;
+
+use Net::Async::Blockchain::BTC;
+
+my $loop = IO::Async::Loop->new;
+
+$loop->add(
+    my $btc_client = Net::Async::Blockchain::BTC->new(
+        subscription_url         => "tcp://127.0.0.1:28332",
+        rpc_url                  => 'http://test:test@127.0.0.1:8332',
+        subscription_timeout     => 100,
```

Comment:
Better to mention the metrics here (+ a short description), at first this looked like seconds to me, until I read the next line.